_posts/2025-02-27-use-desctop-visual-studio-code.adoc

```diff
@@ -0,0 +1,108 @@
+---
+title: "Develop in local Visual Studio Code using Eclipse Che as a remote Cloud Development Environment"
```

Comment:
how about "Working locally in the cloud: Develop in local Visual Studio Code with Eclipse Che as your Cloud Development Environment" as the title?

Reply:
nice, really like the catchy "Working locally in the cloud" pitch 👍

Reply:
@deerskindoll I've changed the title, thanks!